api/src/main/java/ai/djl/nn/AbstractSymbolBlock.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ * with the License. A copy of the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+ * OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+package ai.djl.nn;
+
+import ai.djl.ndarray.NDManager;
+import ai.djl.ndarray.types.Shape;
+
+/** {@code AbstractSymbolBlock} is an abstract implementation of {@link SymbolBlock}. */
+public abstract class AbstractSymbolBlock extends AbstractBlock
```

Comment:
I think Autoloseable should be implemented at subclass level. Not all AbstractSymbolBlock need close()